### PR TITLE
Gallery

### PR DESCRIPTION
**Pull request by prayansh**

### CODE REVIEW

File: css/grayscale.css [hunk 1]
```diff
@@ -273,7 +283,7 @@ ul.nav.navbar-nav {
 .btn-circle:focus {
     outline: 0;
     color: #fff;
-    background: rgba(255,255,255,.1);
+    background: rgba(255, 255, 255, .1);
```
Comment (maiahta) [inline css/grayscale.css:286]: nitpick, but general guideline in front end code is to have a space after a comma and/or colon

File: index.html [hunk 2]
```diff
@@ -168,6 +200,15 @@ <h1 class="brand-heading">
...
 
+
+
+
+
+
+
+
+
+
```
Comment (maiahta) [inline index.html:211]: please remove all this extra whitespace

File: index.html [hunk 3]
```diff
@@ -247,19 +288,53 @@ <h1 class="brand-heading">
...
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>
+        <div class="col-md-4 .col-xs-12">
+            <img class="img-responsive" src="http://placehold.it/300x200" alt="">
+        </div>-->
```
Comment (maiahta) [inline index.html:335]: please remove all unnecessary code

File: index.html [hunk 4]
```diff
@@ -358,6 +433,33 @@ <h3 class="faq__title">FAQ</h3>
...
 <script src="js/grayscale.js"></script>
 
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+<script type="text/javascript" src="http://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+<script type="text/javascript"
+        src="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.pack.min.js"></script>
+<script type="text/javascript">
+    $(function ($) {
+        var addToAll = false;
+        var gallery = true;
```
Comment (maiahta) [inline index.html:443]: correct me if im wrong, but these values seem to be static. why use them at all if they will always remain the same value?

File: index.html [hunk 5]
```diff
@@ -30,7 +30,36 @@
...
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <style type="text/css">
+        a.fancybox img {
+            border: none;
+            -o-transform: scale(1, 1);
+            -ms-transform: scale(1, 1);
+            -moz-transform: scale(1, 1);
```
Comment (maiahta) [inline index.html:43]: -moz-, -ms-, and -o- are for very old versions of their respective browsers. you dont need to include them at all.

File: index.html [hunk 6]
```diff
@@ -358,6 +433,33 @@ <h3 class="faq__title">FAQ</h3>
 <script src="js/AgdaGlobalScripts.js"></script>
 <script src="js/grayscale.js"></script>
 
+<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
+<script type="text/javascript" src="http://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
+<script type="text/javascript"
+        src="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.pack.min.js"></script>
+<script type="text/javascript">
```
Comment (maiahta) [inline index.html:440]: if possible, please move this code to a js file instead of it being written inside the html code

File: index.html [hunk 7]
```diff
@@ -30,7 +30,36 @@
     <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
     <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
     <![endif]-->
+    <!-- For Lightbox Effect-->
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <link rel="stylesheet" type="text/css" media="screen"
+          href="http://cdnjs.cloudflare.com/ajax/libs/fancybox/1.3.4/jquery.fancybox-1.3.4.css"/>
+    <style type="text/css">
```
Comment (maiahta) [inline index.html:38]: if possible, please move this code to a css file instead of it being written inside the html code

File: index.html [hunk 8]
```diff
@@ -358,6 +433,33 @@ <h3 class="faq__title">FAQ</h3>
...
+            var $this = $(this);
+            var title = $this.attr('title');
+            var src = $this.attr('data-big') || $this.attr('src');
+            var a = $('<a href="#" class="fancybox"></a>').attr('href', src).attr('title', title);
+            $this.wrap(a);
+        });
+        if (gallery)
+            $('a.fancybox').attr('rel', 'fancyboxgallery');
+        $('a.fancybox').fancybox({
+            titlePosition: titlePosition
```
Comment (maiahta) [inline index.html:455]: this can be shortened from { titlePosition: titlePosition } to { titlePosition }

**Comment by maiahta** [comment]
Other than the nitpicky issues, after checking out this branch and opening up the page, the gallery is not optimized for mobile which is where we estimate most of the traffic will be going to.
also, the frames are empty, which obviously cannot be merged until filled with the proper pictures.
the most important part, please pull the latest updates off of master. It contains the new pictures, new js, and new css code.

File: index.html (hunk 3 above)
Comment (prayansh) [inline index.html:335]: its not unnecessary per se, its basically boilerplate code for additional images

File: index.html (hunk 4 above)
Comment (prayansh) [inline index.html:443]: true. I apologize for that, I just reused some old code.

File: index.html (hunk 5 above)
Comment (prayansh) [inline index.html:43]: Just trying to cover my bases

File: index.html (hunk 6 above)
Comment (prayansh) [inline index.html:440]: Noted

File: index.html (hunk 7 above)
Comment (prayansh) [inline index.html:38]: Noted

**Pull request closed without merging**